CHANGELOG.md

```diff
@@ -46,6 +46,8 @@ and this project adheres to
   - [#1567](https://github.com/iovisor/bpftrace/pull/1567)
 - Add basic options and documentations for fuzzing
   - [#1601](https://github.com/iovisor/bpftrace/pull/1601)
+- Add speed comparison script betweeen two bpftrace builds
```

Comment:
Dont think a changelog item is needed, its not relevant to end users.

Reply:
I see, removed it.